.drone.yml

```diff
@@ -3,24 +3,26 @@ name: build image
 
 steps:
 - name: build image
-  image: vmck/vmck:0.0.2
+  image: hashicorp/packer:1.4.2
   privileged: true
   volumes:
   - name: vmck-images
-    path: /vmck-images
+    path: /drone/src/packer-build/vmck-images
   commands:
-  - IMAGE_NAME=imgbuild-$(echo $DRONE_COMMIT_BRANCH | tr -s '/' '_').qcow2
-  - SCRIPT=/drone/src/scripts/$(echo $DRONE_COMMIT_BRANCH | cut -f1 -d'/').sh
-  - ARCHIVE_TMP=/vmck-images/$IMAGE_NAME__tmp.tar.gz
-  - ARCHIVE=/vmck-images/$IMAGE_NAME.tar.gz
-  - env
-  - ls -al $SCRIPT
-  - mkdir /tmp/img
-  - /opt/vmck/contrib/build.py /tmp/$IMAGE_NAME --script $SCRIPT
-  - tar czvf $ARCHIVE_TMP -C /tmp $IMAGE_NAME
+  - apk add --update libvirt-daemon qemu qemu-system-x86_64 qemu-img sshpass openssh
```

Comment:
Is `libvirt-daemon` required? Also, doesn't `qemu-system-x86_64` depend on `qemu`?

Reply:
`libvirt-daemon` is the alpine's package for the ubuntu's `qemu-kvm`.

Reply:
I was afraid you'd say that. Ah well.